### ISSUE TITLE
is it support draftjs plugin

### ISSUE THREAD

**Issue by gmchaturvedi1** [globocom/react-native-draftjs-render#18]
I am new to this plugin we create editor with help of draftjs plugin .can we render those with rhis plugin.

**Comment by darlenedms** [comment]
Hi, @gmchaturvedi1

**TL;DR**: We don't support [draft-js-plugins](https://github.com/draft-js-plugins/draft-js-plugins) special types (emoji, stickers, hashtags,...) yet.

**Long Version**: We've developed this lib to render content created using [Megadraft](https://github.com/globocom/megadraft), so for now we only support draft-js default types and provide a way to implement `atomic` types using an `atomicHandler` function (example [here](https://github.com/globocom/react-native-draftjs-render/blob/master/sample/src/index.js#L93)), because that's how [Megadraft](https://github.com/globocom/megadraft) plugins work, e.g.:

```
{
    "key": "3tvln",
    "text": "",
    "type": "atomic",
    "depth": 0,
    "inlineStyleRanges": [],
    "entityRanges": [],
    "data": {
        "type": "image",
        "url": "https://lorempixel.com/400/200/"
    }
}
```

[draft-js-plugins](https://github.com/draft-js-plugins/draft-js-plugins) don't use `atomic` to create plugins, they add new types to draftjs model, e.g.:
```
{
    "key": "7m54a",
    "text": "",
    "type": "sticker",
    "depth": 0,
    "inlineStyleRanges": [],
    "entityRanges": [{
        "offset": 0,
        "length": 0,
        "key": 0
    }],
    "data": {}
}
```

And we don't support it yet :(

**Comment by scf4** [comment]
@darlenedms I haven't taken a look at the code yet, but I'm wondering how much work would need to go into getting it to recognize additional types?

**Comment by Canedo** [comment]
@scf4 Hello! It's not that hard to support new types. 

Draft.js and [megadraft ](https://github.com/globocom/megadraft)have this concept of default types and everything else is an atomic type, but we could actually improve rn-draftjs-render to accept any amount of types.

You can see here:

https://github.com/globocom/react-native-draftjs-render/blob/master/src/getBlocks.js#L75

Right now, there are the basic types, some have specifics logic involved and there is the atomic one, that receives the atomicHandler and the magic happens.

So, if you are willing to submit a PR, I don't think you would find any problem.

**Comment by philohelp** [comment]
A quick update on this topic : it is now possible to use a customBlockHandler exactly in the same way as the atomicHandler function. This greatly simplifies compatibility. Great project made even greater !
